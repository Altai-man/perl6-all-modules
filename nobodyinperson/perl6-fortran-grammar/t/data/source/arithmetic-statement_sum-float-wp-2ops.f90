1.23_wp + 3.456_wp
