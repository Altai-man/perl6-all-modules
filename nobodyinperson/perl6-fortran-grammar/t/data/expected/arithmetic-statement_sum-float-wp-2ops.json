{
  "arithmetic-operator": [
    "+"
  ],
  "value-returning-code": [
    {
      "in-place": {
        "atomic": {
          "number": {
            "float": {
              "digits": [
                "1",
                "23"
              ],
              "precision-spec": {
                "name": "wp"
              }
            }
          }
        }
      }
    },
    {
      "in-place": {
        "atomic": {
          "number": {
            "float": {
              "digits": [
                "3",
                "456"
              ],
              "precision-spec": {
                "name": "wp"
              }
            }
          }
        }
      }
    }
  ]
}
